project/src/components/private-route/private-route.tsx

```diff
@@ -0,0 +1,20 @@
+import {Navigate} from 'react-router-dom';
+import { AppRoute, AuthorizeStatus } from '../../const';
+
+type PrivateRouteProps = {
+  authorizeStatus: AuthorizeStatus;
+  children: JSX.Element;
+}
+
+const PrivateRoute = (props: PrivateRouteProps): JSX.Element => {
```

Comment:
Пропсы можно сразу достать и лучше придерживаться такого подхода далее
```suggestion
const PrivateRoute = ({authorizeStatus, children}: PrivateRouteProps): JSX.Element => {
```